source/_components/rainbird.markdown

```diff
@@ -27,8 +27,30 @@ To enable it, add the following to your `configuration.yaml` file:
 ```yaml
 # Example configuration.yaml entry
 rainbird:
-  host: IP_ADDRESS_OF_MODULE
-  password: YOUR_PASSWORD
+  - host: IP_ADDRESS_OF_MODULE
+    password: YOUR_PASSWORD
+    trigger_time: 6
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Reply:
OK, simplified and added full configuration example in extra block.
Thank you for hint.